pkg/cli/kube.go

```diff
@@ -51,6 +53,10 @@ func (s *Kube) Run(cmd *cobra.Command, args []string) error {
 		return os.WriteFile(s.WriteFile, data, 0644)
 	}
 
+	if err = cobra.MinimumNArgs(1)(cmd, args); err != nil {
+		return err
+	}
```

Comment:
nit:
```suggestion
	if len(args) == 0 {
		return fmt.Errorf("command required when not using '-w' flag")
	}
```

Reply:
Split the difference and did an error wrap